common/api-review/auth.api.md

```diff
@@ -165,6 +165,7 @@ export const AuthErrorCodes: {
     readonly INVALID_EMAIL: "auth/invalid-email";
     readonly INVALID_EMULATOR_SCHEME: "auth/invalid-emulator-scheme";
     readonly INVALID_IDP_RESPONSE: "auth/invalid-credential";
+    readonly INVALID_LOGIN_CREDENTIALS: "auth/invalid-credential";
```

Comment:
invalid-login-credential?

Reply:
auth/invalid-credential error code already exists. I am reusing the same one here rather than introduce a new error code "auth/invalid-login-credential" which is very similar.

Reply:
How do we map the backend errors then?

Reply:
In line 150 in packages/auth/src/api/errors.ts

Reply:
Okay, I think there could be potential problems like `ServerError.INVALID_LOGIN_CREDENTIALS` being mapped to `AuthErrorCode.INVALID_IDP_RESPONSE`, since there underlying values are the same.

I suggest use unique enum values unless there are clear benefits.

Reply:
The benefit is mainly to reduce introducing new error codes that are very similar to existing error codes. auth/invalid-credential and auth/invalid-login-credential look almost identical and it is not immediately obvious how they are different.

But the multiple enums with the same value seem risky, example - https://stackoverflow.com/a/58644229

Will add a new error code.